deploy/setup-project.sh

```diff
@@ -1,13 +1,26 @@
 #!/bin/bash
 
+# This script will setup the given project with a Service Account that has the correct
+# restricted permissions to run the gcp_compute_persistent_disk_csi_driver and download
+# the keys to a specified directory
+
+# WARNING: This script will delete and recreate all service accounts, bindings, and keys.
```

Comment:
ALL service accounts???

Reply:
ohoh thats alarming. No just the one set.